bin/eslint.js

```diff
@@ -9,6 +9,11 @@
 
 "use strict";
 
+const mod = require("node:module");
+
+// to use V8's code cache to speed up instantiation time
+mod.enableCompileCache?.();
+
```

Comment:
Would it work for developers using the Node.js API and not CLI?

Reply:
it's intentional - see https://github.com/eslint/eslint/pull/11921#pullrequestreview-256057049

Reply:
Makes sense 👍🏻 .